scss/components/_case-list-table.scss

```diff
@@ -51,7 +51,7 @@
 
     th:first-child {
       background: $gray-lighter;
-      height: 61px;
+      height: 63px;
```

Comment:
Any reason for this magic number. Can't we accommodate the height using padding and line-heights?

Reply:
I just updated the old one, as the UI was broken. Looks like, if I dont mention the height here, it looks fine. will do that. thanks